app/vendors/vendor-dashboard/testimonials/page.tsx

```
@@ -0,0 +1,149 @@
+              <CardContent>
+                <TestimonialsList testimonials={vendor.testimonials} />
+              </CardContent>
+            </Card>
+          </TabsContent>
+
+          <TabsContent value="request" className="mt-6">
+            <Card>
+              <CardHeader>
+                <CardTitle>Request a Testimonial</CardTitle>
+                <CardDescription>
+                  Send a testimonial request to a candidate you've worked with
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)